specification/use_case_library.md

```diff
@@ -5,42 +5,42 @@ The following table contains a set of commonly performed FinOps scenarios that w
+| FinOps Practitioner | FinOps & Intersecting Frameworks | As a FinOps Practitioner, I need to verify the accuracy of the cloud service provider invoices and the underlying services | ProviderName<br>BillingAccountID<br>BillingAccountName<br>BillingPeriodStart<br>BilledCost<br>BillingCurrency<br>ServiceName |
+| FinOps Practitioner | FinOps & Intersecting Frameworks | As a FinOps Practitioner, I need to reconcile discounts on the cloud service provider invoices and the underlying services | ProviderName<br>BillingAccountId<br>BillingAccountName<br>BillingPeriodStart<br>BilledCost<br>BillingCurrency<br>EffectiveCost<br>ListCost<br>ServiceName |
+| FinOps Practitioner | FinOps & Intersecting Frameworks | As a FinOps Practitioner, I need to analyze usage data of resources | ChargePeriodStart<br>ChargeCategory<br>EffectiveCost<br>ProviderName<br>PricingQuantity<br>ConsumedQuantity<br>ResourceId<br>ServiceName<br>SKUId<br>UsageUnit |
+| FinOps Practitioner | Forecasting | As a FinOps Practitioner, I need to forecast costs, based on historical usage trends and rates | BillingPeriodStart<br>ChargeType<br>ChargeDescription<br>EffectiveCost<br>ProviderName<br>PricingQuantity<br>ConsumedQuantity<br>Region<br>ServiceCategory<br>ServiceName<br>SKUId<br>UsageUnit |
+| FinOps Practitioner | Managing Anomalies | As a FinOps Practitioner, I need to see the daily costs across all cloud providers, billing accounts, and sub accounts | BillingAccountId<br>SubAccountId<br>ChargePeriodStart<br>ChargePeriodEnd<br>ProviderName<br>EffectiveCost |
+| FinOps Practitioner | Managing Anomalies | As a FinOps Practitioner, I need to see the daily costs across all cloud providers, billing accounts, sub accounts, and region | BillingAccountId<br>SubAccountId<br>ChargePeriodStart<br>ChargePeriodEnd<br>EffectiveCost<br>ProviderName<br>Region |
+| FinOps Practitioner | Managing Anomalies | As a FinOps practitioner, I need to see the daily costs across all cloud providers, billing accounts, sub accounts, and service | BillingAccountId<br>SubAccountId<br>ChargePeriodStart<br>ChargePeriodEnd<br>EffectiveCost<br>ProviderName<br>ServiceName |
+| FinOps Practitioner | Managing Commitment Based Discounts | As a FinOps Practitioner, I want to track all commitment based discounts purchased for a time period | ProviderName<br>BillingAccountId<br>CommitmentDiscountId<br>CommitmentDiscountType<br>BilledCost<br>ChargePeriodStart<br>ChargeCategory |
 | FinOps Practitioner | Managing Commitment Based Discounts | As a FinOps Practitioner, I want to track unused commitment charges in any given time period so that I consider them in my future commitment planning or remedy them | CommitmentDiscountStatus (filter)<br>CommitmentDiscountID<br>BilledCost<br>ChargePeriodStart |
```

Comment:
```suggestion
| FinOps Practitioner | Managing Commitment Based Discounts | As a FinOps Practitioner, I want to track unused commitment charges in any given time period so that I consider them in my future commitment planning or remedy them | CommitmentDiscountStatus (filter)<br>CommitmentDiscountId<br>BilledCost<br>ChargePeriodStart |
```